pkg/v22/resource/workercount/create.go

```diff
@@ -0,0 +1,66 @@
+package workercount
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/errors/tenant"
+	"github.com/giantswarm/microerror"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/cluster-operator/pkg/label"
+	"github.com/giantswarm/cluster-operator/pkg/v22/controllercontext"
+	"github.com/giantswarm/cluster-operator/pkg/v22/key"
+)
+
+func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	cr, err := r.toClusterConfigFunc(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+	cc, err := controllercontext.FromContext(ctx)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	{
+		if cc.Client.TenantCluster.K8s == nil {
+			r.logger.LogCtx(ctx, "level", "debug", "message", "tenant cluster clients not available yet")
+			r.logger.LogCtx(ctx, "level", "debug", "message", "canceling resource")
```

Comment:
Don't we need to add a cancellation context here?

Reply:
No that is only needed for CRUD resources.